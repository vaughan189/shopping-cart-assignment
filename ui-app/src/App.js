import React from "react";

import { BrowserRouter as Router } from "react-router-dom";
// import { Footer } from "./components/Footer";
import { Header } from "./components/Header";

import { Routes } from "./routes";

function App() {
  return (
    <Router>
      <React.StrictMode>
        <Header />
        <div>
          <Routes />
        </div>
        {/* <Footer /> */}
      </React.StrictMode>
    </Router>
  );
}

export default App;
