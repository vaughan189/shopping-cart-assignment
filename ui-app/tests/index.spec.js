import React from "react";
import { shallow } from "enzyme";
import App from "../src/App";

describe("App component", () => {
  it("should render the App component correctly", () => {
    const component = shallow(<App debug />);
    expect(component).toMatchSnapshot();
  });
});
