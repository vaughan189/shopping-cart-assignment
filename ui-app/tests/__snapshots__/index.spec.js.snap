// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render the App component correctly 1`] = `
<BrowserRouter>
  <Header />
  <div>
    <Routes />
  </div>
  <Footer />
</BrowserRouter>
`;
